### PR TITLE
examples: enable building more examples with ch-image

### PR DESCRIPTION
**Pull request by heasterday**
Addresses #883 and #892

### CODE REVIEW

**Comment by heasterday** [comment]
Tested locally and on CTS-1.

File: examples/lammps/Dockerfile
Comment (j-ogas) [inline examples/lammps/Dockerfile:2]: Is this going to be merged before #905 ?

Comment (reidpr) [reply examples/lammps/Dockerfile:2]: No; but we didn't write that down.

**Comment by reidpr** [comment]
Per discussion offline we are going to wait until PR #905 is merged, because then this PR can close more issues with minimal additional effort.

File: lib/fakeroot.py
Comment (reidpr) [inline lib/fakeroot.py:195]: Copy-edit, including line wrapping:

```suggestion
   # 1. In recent Debian-based distributions apt(8) runs as an unprivileged
   #    user by default. This makes *all* apt operations fail in an
   #    unprivileged container because it can't drop privileges. There are
   #    multiple ways to turn the “sandbox” off. AFAICT, none are documented,
   #    but this one at least appears in Google searches a lot.
```

File: lib/fakeroot.py
Comment (reidpr) [inline lib/fakeroot.py:214]: Can you:
1. adjust the key to reflect the new Ubuntu inclusion, and
2. revise the name to meet the character limit (see long tedious comment at the beginning of `DEFAULT_CONFIGS`

**Review by reidpr — CHANGES_REQUESTED** [review]
Thanks, a few minor issues. Let's defer it until 0.22 to relieve the pressure since it's not high priority.

**Comment by reidpr** [comment]
Oh, also please add tests in `50_fakeroot.bats` for the Ubuntu stuff. I see there's 3 distros added so not 100% sure what the right coverage is.

File: lib/fakeroot.py
Comment (reidpr) [inline lib/fakeroot.py:211]:
```suggestion
   "debian":
```

Comment (reidpr) [inline lib/fakeroot.py:211]:
```suggestion
   "debSBubXBF":
```

File: lib/fakeroot.py
Comment (reidpr) [inline lib/fakeroot.py:213]:
```suggestion
   { "name": "Debian (9, 10) or Ubuntu (16, 18, 20)",
     "match": ("/etc/os-release", r"(stretch|buster|xenial|bionic|focal)"),
```

File: test/build/50_fakeroot.bats
Comment (reidpr) [inline test/build/50_fakeroot.bats:469]: Can we add the version numbers too and put them in order by version?

**Review by reidpr — CHANGES_REQUESTED** [review]
Nice work! A few minor nitpicks.

File: lib/fakeroot.py
Comment (reidpr) [reply lib/fakeroot.py:211]: I'm not 100% sold on these but those are the suggestions I had. Maybe another is `debderiv`.

Comment (heasterday) [reply lib/fakeroot.py:211]: I like `debderiv` I think

**Comment by heasterday** [comment]
@reidpr Changes made, let me know if there is anything else.

**Comment by heasterday** [comment]
Passes tests locally and on GPU enabled system.

**Pull request merged**
